src/rust/cxx-integration/lib.rs

```diff
@@ -0,0 +1,22 @@
+pub mod tokio;
+
+#[cxx::bridge(namespace = "workerd::rust::cxx_integration")]
+mod ffi {
+    extern "Rust" {
+        fn init();
+
+        fn trigger_panic(msg: &str);
+    }
+
+    unsafe extern "C++" {
+        include!("rust/cxx-integration/cxx-bridge.h");
+    }
+}
+
+pub fn init() {
+    tokio::init();
```

Comment:
Do we plan to be using tokio soon within workerd? If it's only inited here and used in tests but not prod code it might be better to start depending on it later on

Reply:
I want us to have the same set up so that there's less friction writing/moving code. E.g. if one decided upstream log fowarding or something :)